packages/web-components/src/components/va-button-icon/va-button-icon.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  Component,
+  Event,
+  EventEmitter,
+  Host,
+  h,
+  Listen,
+  Prop,
+  Element,
+} from '@stencil/core';
+import classnames from 'classnames';
+
+/**
+ * @componentName Button - Icon
+ * @nativeHandler onClick
+ * @maturityCategory caution
+ * @maturityLevel candidate
+ */
+
+@Component({
+  tag: 'va-button-icon',
+  styleUrl: 'va-button-icon.scss',
+  shadow: true,
+})
+export class VaButtonIcon {
+  @Element() el: HTMLElement;
+
+  /**
+   * If `true`, the component-library-analytics event is disabled.
+   */
+  @Prop() disableAnalytics?: boolean = false;
+
+  /**
+   * If `true`, the click event will not fire.
+   */
+  @Prop({ reflect: true }) disabled?: boolean = false;
```

Comment:
I just want to double check that we want a `disabled` prop? I know are some usability issues with offering that as an option and I didn't see a disabled state in the Figma file for this.

Reply:
Confirmed with Matt Dingee that we don't want a `disabled` state.